stubs/click-web/click_web/resources/cmd_exec.pyi

```diff
@@ -0,0 +1,79 @@
+import logging
+
+from flask import Response
+
+from .input_fields import FieldId
+
+logger: logging.Logger | None
+
+HTML_HEAD: str
+HTML_TAIL: str
+
+class Executor:
+    RAW_CMD_PATH: str
+
+    def __init__(self) -> None: ...
+    def exec(self, command_path: str) -> Response: ...
+    def _exec_raw(self, command: list[str]) -> Response: ...  # undocumented
```

Comment:
if it's private (single underscore) + undocumented we probably should leave it out unless it's specifically requested for